bash_completion

```diff
@@ -41,18 +41,24 @@ fi
 shopt -s extglob progcomp
 
 # Declare a compatibility function name
-# @param $1 Old function name
-# @param $2 New function name
+# @param $1 Version of bash-completion where the deprecation occurred
+# @param $2 Old function name
+# @param $3 New function name
+# @since 2.12
 _comp_deprecate_func()
 {
-    if [[ $1 != [a-zA-Z_]*([a-zA-Z_0-9]) ]]; then
-        printf 'bash_completion: %s: %s\n' "$FUNCNAME" "\$1: invalid function name '$1'" >&2
+    if (($# != 3)); then
+        printf 'bash_completion: %s: usage: %s OLD_NAME NEW_NAME DEPRECATION_VERSION\n' "$FUNCNAME" "$FUNCNAME"
+        exit 2
```

Comment:
```suggestion
        return 2
```

I think `exit` would terminate the shell so we cannot start an interactive session with a mistake.

Reply:
Oops, indeed, thanks!